webpack/ForemanTasks/Components/TasksTable/TasksTable.js

```diff
@@ -18,7 +18,6 @@ const TasksTable = ({
   results,
   history,
   itemCount,
-  pagination,
```

Comment:
can we also remove `pagination` and its selector from the `index.js` wrapper?

Reply:
The component still needs the perpage so change the selector from pagination to perpage (for selectAll functionality)